### PR TITLE
CSVUploadTest: transform expectedDate to server TZ

### PR DESCRIPTION
**Pull request by rmakulov**

### CODE REVIEW

File: src/test/java/com/axibase/tsd/api/method/version/VersionMethod.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package com.axibase.tsd.api.method.version;
+
+import com.axibase.tsd.api.method.BaseMethod;
+
+import javax.ws.rs.core.Response;
+
+public class VersionMethod extends BaseMethod {
```
Comment (Sild) [inline src/test/java/com/axibase/tsd/api/method/version/VersionMethod.java]: should we leave it here or move to BaseMethod?

Comment (rmakulov) [inline src/test/java/com/axibase/tsd/api/method/version/VersionMethod.java]: Let's move.

**Comment by Sild** [comment]
review done

**Comment by Sild** [comment]
ok

**Pull request merged**